azure-functions/acbs-function/mappings/facility/facility-loan.js

```diff
@@ -0,0 +1,74 @@
+/*
+"portfolioIdentifier":              Facility portfolio idenfifier. Default to "E1"
+"postingDate":                      Date of submission to ACBS
+"facilityIdentifier":               UKEF facilityId
+"borrowerPartyIdentifier":          exporter ACBS ID
+"productTypeId":                    Facility Type i.e. 250 for BOND
+"productTypeGroup":                 Product group EWCS (EW) , Bond (BS) and GEF (GM)
+"currency":                         Facility currency code
+"dealCustomerUsageRate":            This is Currency exchange rate (FX rate)
+"dealCustomerUsageOperationType":   This is Currency exchange rate operation (D for divide, M for Multiply), set to `D`
+"amount":                           ukefExposure
+"issueDate":                        At Commitment stage its not required ,
+                                    Issued when issued it would be Issue Date for Bond OR Disbursement Date for Loan
+"expiryDate":                       Facility expiry date
+"spreadRate":                       Used for Premium Accrual Schedule. For GEF, BSS and EWCS.
+"spreadRateCTL":                    Used for Contractual Interest Accrual Schedule (CTL).
+                                    For EWCS. EWCS needs two spread rates.
+"nextDueDate":
+"yearBasis":                        dayCountBasis : 360 = 5, 365 = 1
+"indexRateChangeFrequency":         feeFrequency
+"loanBillingFrequencyType":         feeType
+*/
+
+const helpers = require('./helpers');
+const date = require('../../helpers/date');
+const CONSTANTS = require('../../constants');
+const getDealSubmissionDate = require('../deal/helpers/get-deal-submission-date');
+
+const facilityLoan = (deal, facility, acbsData) => {
+  const issueDate = helpers.getIssueDate(facility, getDealSubmissionDate(deal));
+  const { guaranteeExpiryDate } = facility.tfm.facilityGuaranteeDates
+    ? facility.tfm.facilityGuaranteeDates
+    : '';
+
+  let loanRecord = {
+    portfolioIdentifier: CONSTANTS.FACILITY.PORTFOLIO.E1,
+    postingDate: date.now(),
```

Comment:
Does this work? Or do we need to use `Date.now()` ?

Reply:
This does it is method from `date.js` helpers collection file.